### PR TITLE
add set/get cache size function for HumanMonitor and QMPMonitor

### PR DESCRIPTION
**Pull request by XueqiangWei**
Signed-off-by: Xueqiang Wei <xuwei@redhat.com>

### CODE REVIEW

**Comment by XueqiangWei** [comment]
ID 1342353

**Comment by xutian** [comment]
Acked, thanks @XueqiangWei

File: virttest/qemu_monitor.py [hunk 1]
```diff
@@ -1239,6 +1239,25 @@ def get_migrate_capability(self, capability):
...
+        self.verify_supported_cmd(cmd)
+        cmd += " %s" % value
+        return self.cmd(cmd)
+
+    def get_migrate_cache_size(self):
+        """
+        Get the xbzrel cache size. e.g. xbzrel cache size: 1048576 kbytes
+        """
+        cache_size_info = self.query("migrate_cache_size")
+        value = cache_size_info.split(":")[1].split()[0].strip()
```
Comment (CongLi) [inline virttest/qemu_monitor.py]: does it need convert type here?

Comment (XueqiangWei) [inline virttest/qemu_monitor.py]: thanks coli, it doesn't need convert type, str is ok. easy to compare with the set value by funtion "set_migrate_cache_size"

**Comment by CongLi** [comment]
Acked-by: Cong Li <coli@redhat.com>

**Comment by xutian** [comment]
merged, thanks @XueqiangWei and @CongLi

**Pull request merged**